hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeCreateRequest.java

```diff
@@ -71,10 +71,12 @@ public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
     VolumeInfo volumeInfo  =
         getOmRequest().getCreateVolumeRequest().getVolumeInfo();
 
-    // Set creation time
+    // Set creationTime and modificationTime
     VolumeInfo updatedVolumeInfo =
-        volumeInfo.toBuilder().setCreationTime(Time.now()).build();
-
+        volumeInfo.toBuilder()
+        .setCreationTime(Time.now())
```

Comment:
Here do we need to get the value once, and set the same value for creatiionTime and ModificationTime. Because once after createBucket, there is a chance of these having different values.

long updateTime = Time.now(), and then use this in setting mofication and creation time.

Reply:
Thanks @bharatviswa504 for the comment.
Fixed.